apps/src/sites/studio/pages/regional_partners/regional_partner_search.js

```diff
@@ -7,14 +7,18 @@ import {getStore, registerReducers} from '@cdo/apps/redux';
 import isRtl from '@cdo/apps/code-studio/isRtlRedux';
 import responsive from '@cdo/apps/code-studio/responsiveRedux';
 import initResponsive from '@cdo/apps/code-studio/responsive';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
 
 registerReducers({isRtl, responsive});
 
 $(document).ready(initRegionalPartnerSearch);
 
 function showRegionalPartnerSearch() {
   const regionalPartnerSearchElement = $('#regional-partner-search');
-  const sourcePageId = regionalPartnerSearchElement.data('source-page-id');
+  const sourcePageId = 'program-information';
```

Comment:
Does this mean this will be the only place the regional partner search is used?

Reply:
Yep!
```
bethanys-mbp:code-dot-org bethanyconnor$ git grep -l 'RegionalPartnerSearch'
apps/src/sites/studio/pages/regional_partners/regional_partner_search.js
apps/src/templates/RegionalPartnerSearch.jsx
apps/test/unit/templates/RegionalPartnerSearchTest.js
```

`sourcePageId` is sent back to the server to be used in some logging and I think the route that requires it is used in multiple places, so I didn't want to completely clean it up. But figured it didn't really need to be passed down from the server 🤷‍♀️

Reply:
Gotcha that makes sense!